packages/bpk-component-popover/stories.js

```diff
@@ -141,25 +147,34 @@ PopoverContainer.defaultProps = {
   targetFunction: null,
 };
 
+const Spacer = props => (
+  <div
+    className={getClassName('bpk-popover-spacer')}
+    {...props}
+  />
+);
+
 storiesOf('bpk-component-popover', module)
   .add('Default', () => (
-    <div style={{ height: '1000px', margin: '30px', textAlign: 'center' }}>
-      <PopoverContainer />
-    </div>
+    <Spacer>
+      <PopoverContainer id="my-popover-1" />
+      <PopoverContainer id="my-popover-2" />
```

Comment:
Is the addition of `id` something that the new `eslint` brought in?

Reply:
Nah I did that because there are two on the page and I wanted them to have different `id`'s